nf_core/lint/multiqc_config.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import os
+import yaml
+
+
+def multiqc_config(self):
+    """Make sure basic multiQC plugins are installed and plots are exported
+    Basic template:
+    report_comment: >
+        This report has been generated by the <a href="https://github.com/nf-core/quantms" target="_blank">nf-core/quantms</a>
+        analysis pipeline. For information about how to interpret these results, please see the
+        <a href="https://nf-co.re/quantms" target="_blank">documentation</a>.
+    report_section_order:
+        software_versions:
+            order: -1000
+        nf-core-quantms-summary:
+            order: -1001
+
+    export_plots: true
```

Comment:
Please format this as an RST code block. eg: https://github.com/nf-core/tools/blob/b1ce47148a05965e0604534cece3359d408af357/nf_core/lint/actions_ci.py#L19-L27

Reply:
Do you think this is even needed here? It was mostly for me to check what should be there.

Reply:
Yeah maybe not, I guess it's at fairly high risk of drifting if the pipeline changes. Maybe just link to the template file?

Reply:
meh, it'll do for now. Added as a code block in 57ca562136d882740dc2afd4901a41725838cea4